index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Source Code Pro' rel='stylesheet'> 
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!-- <h1>Hello World!, we are software developers!</h1> -->
+    <script  src="js/index.js"></script>
+    <div id="title-header" style="margin-left: 350px;
+    font-size: 20px; padding: 25px; color: #E900FF;">
+      <h1>JavaScript reference</h1>
+    </div>
+    <nav id="navbar">
+      <header id="nav-header">
+        <h2>JS documentation</h2>

```

Comment:
I'd suggest that the header should contain the nav-bar, and not the other way around